package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.7.1",
```

Comment:
why do we need these packages `"@emotion/react"` and `"@emotion/styled"`

Reply:
These are required as part of installing material ui